From: Richard B Winters <rik@mmogp.com>
Date: Tue, 21 Apr 2015 20:42:12 -0400
Subject: Remove .hgsigs from source

The .hgsigs file should not be packed with the release tarball.

Change-Id: I72da4087cdb3bb321f5a5a80a5b3758cc488b79a
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 .hgsigs | 4 ----
 1 file changed, 4 deletions(-)
 delete mode 100644 .hgsigs

diff --git a/.hgsigs b/.hgsigs
deleted file mode 100644
index ddb9f3d..0000000
--- a/.hgsigs
+++ /dev/null
@@ -1,4 +0,0 @@
-b6dca003bb176978af803eeb33019b6aef3c58b0 0 iEYEABECAAYFAktnGJAACgkQWnQN9wr0w1ywBACghXYwYkv/70Xg5AQMzVjRWKZecIoAnjRUytRoYl+dhhqbhfdXSD+/Bfvw
-6007185b487e3f2dc3b24674a9105761b2cde6ea 0 iEYEABECAAYFAktoWfsACgkQWnQN9wr0w1ySZwCfQILyC2VJrCnVEU2zvTIyI7ustDAAn37hhb9JM8JQVKLfPEbqIloz1m3m
-979c4ffae287976631a30d10258903aea6fb28fa 0 iEYEABECAAYFAktoY38ACgkQWnQN9wr0w1wHxgCeMZyOt8ccxmIsvIHg4/y6KmqtTVAAn2jn7dOmFMjA8m4ju59YaQ1Bznhb
-798645ead29e20b361af883fce695b85caf3392b 0 iEYEABECAAYFAlPJCM4ACgkQWnQN9wr0w1wv+QCeJQOjaYNXNJZA61n7Fu8f63CeVBEAnja4WqiYC+TS4HvmRJz6oNi6p48u
